src/server/index.tsx

```diff
@@ -26,7 +26,28 @@ server.use(express.json() as RequestHandler);
 server.use(express.urlencoded({ extended: false }) as RequestHandler);
 server.use("/static", express.static(path.resolve("./dist")));
 server.use("/docs", express.static(path.resolve("./dist/assets/docs")));
-server.use("/api", express.static(path.resolve("./dist/assets/api")));
+
+// The latest release docs
+// TODO OpenAPI isn't currently working for the latest docs. This can be changed after the next release.
+// server.use("/api/latest", express.static(path.resolve("./dist/assets/api_latest.html")));
+server.use(
+  "/api/latest",
+  express.static(path.resolve("./dist/assets/lemmy-js-client-latest-docs")),
+);
+server.use(
+  "/lemmy-js-client-docs/latest",
+  express.static(path.resolve("./dist/assets/lemmy-js-client-latest-docs")),
```

Comment:
So for latest, I just duplicate the existing lemmy typedocs at `api/v4`, as well as `/lemmy-js-client-docs/latest`